app/controllers/managers_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class ManagersController < ApplicationController
+  before_action :set_manager, only: %i[show update destroy]
+
+  def index
+    render json: Manager.all
+  end
+
+  def show
+    render json: @manager
+  end
+
+  # POST
+  def create
+    @manager = Manager.new(manager_params)
+    if @manager.save
+      render json: @manager, status: :created, location: @manager
+    else
+      render json: @manager.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT
+  def update
+    if @manager.update(manager_params)
+      render json: @manager
+    else
+      render json: @manager.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE
+  def destroy
+    @manager.destroy
+  end
```

Comment:
This will not return as it doesn't have a render call nor an associated view for the path